src/HomeKit/HMEnums.cs

```diff
@@ -208,7 +208,18 @@ public enum HMCharacteristicType {
 		TargetPosition,
 		[iOS (9,0)]
 		TargetVerticalTilt,
-
```

Comment:
It would be noise if it was unused information, but the IDE can use it :) Our documentation also use this information (so looking up the docs would not help either). Without it the developer needs to track down the field itself (internal) to see it's availability.

It's also removed by the linker so there's no _cost_ in the applications :)